cms/static/sass/elements/_header.scss

```diff
@@ -272,11 +272,11 @@
   .wrapper-header {
 
     .wrapper-l {
-      width: flex-grid(8,12);
+      width: flex-grid(9,12);
     }
 
     .wrapper-r {
-      width: flex-grid(4,12);
+      width: flex-grid(3,12);
```

Comment:
@roderickmorales Do you think these changes are okay, the thing i want to ask in particular is, there 2 different views here: 1) when you're signed in within course 2) when you're signed in but not in course i.e. home page. So currently we made change to `.course`, do you think we should also make this change to non-course view ? Just for consistency.